src/explore-education-statistics-admin/src/pages/admin-dashboard/components/MethodologySummary.tsx

```diff
@@ -96,118 +96,127 @@ const MethodologySummary = ({
     onChangePublication();
   };
 
-  const hasMethodologyPermissions =
+  const canCreateOrManageExternal =
     publication.permissions.canCreateMethodologies ||
     publication.permissions.canManageExternalMethodology;
 
   return (
     <>
       {methodologies.length > 0 &&
-        methodologies.map(methodology => (
-          <Details
-            key={methodology.id}
-            open={false}
-            className="govuk-!-margin-bottom-0"
-            summary={methodology.title}
-            summaryAfter={
-              <TagGroup className="govuk-!-margin-left-2">
-                <Tag>{methodology.status}</Tag>
-                {methodology.amendment && <Tag>Amendment</Tag>}
-              </TagGroup>
-            }
-          >
-            <SummaryList className="govuk-!-margin-bottom-3">
-              <SummaryListItem term="Publish date">
-                {methodology.published ? (
-                  <FormattedDate>{methodology.published}</FormattedDate>
-                ) : (
-                  'Not yet published'
-                )}
-              </SummaryListItem>
-              {methodology.internalReleaseNote && (
-                <SummaryListItem term="Internal release note">
-                  {methodology.internalReleaseNote}
+        methodologies.map(methodology => {
```

Comment:
As we're not doing anything with the inverse of the `methodologies.length > 0`, we could simplify this to just:
```ts
methodolgies.map(methodology => ...
```

If there are no methodologies, nothing would be rendered anyway.

Reply:
Cool, I've removed the length check now.